src/app/tests/suites/TestSystemCommands.yaml

```diff
@@ -79,6 +79,8 @@ tests:
                 value: 5560
               - name: "kvs"
                 value: "/tmp/chip_kvs_default"
+              - name: "minCommissioningTimeout"
```

Comment:
With the new change, probably a good test is to just call `Start` with `minCommissioningTimeout` to show that all the arguments that come before it is not necessary?